### DISCUSSION TITLE
Anycubic Photon Mono X pwmx versions

### DISCUSSION THREAD

**Discussion by tiberiustribun** [sn4k3/UVtools#613] [Q&A] [answered]
I want to use Pusa Slicer files on my Photon Mono X. When I convert the SL1 file in UVtools to the pwmx format for my printer a popup asks me for the correct version out of 1, 515, 516 or 517. Choosing "1" leads to the printer rejecting the file due to unknown print parameter.

Is there documentation somewhere which version of pwmx corresponds to which firmware version on the printer? I searched on google for half an hour and came up with nothing.

**Comment by sn4k3** [comment, marked as answer]
> Is there documentation somewhere which version of pwmx corresponds to which firmware version on the printer? I searched on google for half an hour and came up with nothing.

Welcome to the SLA printing universe, 99.99% of the products have this same rule. They want only to sell toys, everything else is ignored, even if you ask them.

Mono X is capable of v1, but if you upgrade your firmware you should choose v516, however v516 should be retro-compatible with v1 and vice-versa (If made correctly on firmware).

If you are using PrusaSlicer you "must" use my printers profiles and tune from there.

**Comment by tiberiustribun** [comment]
I am still on the original firmware v3.4.6 that came with the printer. So I thought pwmx version 1 should work. When Chitubox switched to the subscription based Pro version I refrained from updates because I thought maybe they might move features to chitubox pro only. It seems thats only the case for newer printers though an maybe I should update finally. 

I don't use the printer very often, the frustrations with the toolchain being one of the major causes for that. I am a Linux user and the amount of acceptable software for these printers is very low. I can say with confidence that without UVtools I would have given up on msla printers a while ago so thank you very much for the good work. 
The slicer file conversion correctly working would be awesome because as of now my toolchain is PrusaSlicer->Chitubox->UVtools and I would very much like to get rid of Chitubox.

**Reply by sn4k3** [reply]
Have you followed this [guide](https://github.com/sn4k3/UVtools/wiki/Setup-PrusaSlicer)?

**Reply by tiberiustribun** [reply]
Not until yesterday but I did everything it said.
I will try updating the printer firmware and start with a fresh version of Prusaslicer and UVtools and going through the guide again.
